tox.ini

```diff
@@ -25,13 +26,23 @@ exclude =
 ignore = # more info: https://www.flake8rules.com/
     D102 # method docstring
     D103 # function docstring
+    D105 # docstring in magic method
     D107 # init docstring
+    D400 # method docstring
```

Comment:
This is first line ending with a period or? Why that has to be ignored?

Reply:
Actually I think we can get rid of the D warnings under flake by just removing flake8-docstring. Let me give that a try

Reply:
`flake8-docstrings` indeed causes these warnings and `pydocstyle` makes them redundant 45e473d9932998ec0b965a3257713db2c6f769b7